relayer/chains/cosmos/msg_handlers_packet_test.go

```diff
@@ -26,9 +26,11 @@ func mockCosmosChainProcessor(t *testing.T) *CosmosChainProcessor {
 		provider      = cosmos.CosmosProvider{PCfg: cosmos.CosmosProviderConfig{ChainID: chainID1}}
 	)
 
-	ccp, err := NewCosmosChainProcessor(log, &provider, "", os.Stdin, os.Stdout, []*processor.PathProcessor{pathProcessor})
+	ccp, err := NewCosmosChainProcessor(log, &provider, "", os.Stdin, os.Stdout)
```

Comment:
This one should avoid `os.Stdin` and `os.Stdout` too.

Reply:
done